plugins/modules/identity/keycloak/keycloak_group_rolemapping.py

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022, Marius Huysamen (@mhuysamen)
```

Comment:
In case you copied an existing module and modified it, you have to keep the original copyright as well.

Reply:
Yes, I used keycloak_user_rolemapping as a starting point.  Will add the original author.

Reply:
The important part isn't the author, but the copyright. You only added the author, but not the copyright.